.github/workflows/upload-deb-images.yml

```diff
@@ -0,0 +1,56 @@
+name: Upload package creation Docker images - DEB - amd64 and i386
+on:
+  pull_request:
+    paths:
+      - 'debs/Debian/**'
+      - 'debs/build.sh'
+  push:
+    branches:
+      - master
+      - 4.5
+      - 4.4
+    paths:
+      - 'debs/Debian/*'
+      - 'debs/build.sh'
+  workflow_dispatch:
+
+jobs:
```

Comment:
Just a question, can the names of the jobs be written with spaces? `Upload deb package building images` instead of `Upload-deb-package-building-images`

Reply:
No, they cannot contain any space